### PR TITLE
added policies

### PR DESCRIPTION
**Pull request by ch-stark**
- improvement of upgrade-policy
- added further policies

### CODE REVIEW

File: community/CM-Configuration-Management/policy-scheduler.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  name: policy-scheduler
+  annotations:
+    policy.open-cluster-management.io/standards: NIST 800-53
+    policy.open-cluster-management.io/categories: CM Configuration Management
+    policy.open-cluster-management.io/controls: CM-2 Baseline Configuration
+spec:
+  remediationAction: inform
+  disabled: false
+  policy-templates:
+    - objectDefinition:
```
Comment (JustinKuli) [inline community/CM-Configuration-Management/policy-scheduler.yaml:13]: Should this objectDefinition be a ConfigurationPolicy which contains the Scheduler?

Comment (ch-stark) [reply community/CM-Configuration-Management/policy-scheduler.yaml:13]: @JustinKuli I added policies based on:  https://docs.openshift.com/container-platform/4.7/post_installation_configuration/cluster-tasks.html#configuration-resources_post-install-cluster-tasks

regarding:
Should this objectDefinition be a ConfigurationPolicy which contains the Scheduler?

yes

**Comment by ch-stark** [comment]
corrected Scheduler-Policy

File: community/CM-Configuration-Management/policy-ingress-controller.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    policy.open-cluster-management.io/controls: CM-2 Baseline Configuration, PR.IP-1 Baseline Configuration
+spec:
+  remediationAction: inform
+  disabled: false
+  policy-templates:
+    - objectDefinition:
+        apiVersion: policy.open-cluster-management.io/v1
+        kind: ConfigurationPolicy
+        metadata:
+          name: policy-autoscaler
```
Comment (gparvin) [inline community/CM-Configuration-Management/policy-ingress-controller.yaml]: I think you meant to name this something related to policy-ingress-controller and not for the other autoscaler policy

File: community/CM-Configuration-Management/policy-ingress-controller.yaml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+kind: PlacementRule
+metadata:
+  name: placement-policy-ingress-controller
+spec:
+  clusterConditions:
+    - status: 'True'
+      type: ManagedClusterConditionAvailable
+  clusterSelector:
+    matchExpressions:
+      - key: env
```
Comment (gparvin) [inline community/CM-Configuration-Management/policy-ingress-controller.yaml]: When possible we want to use `environment` instead of `env`

**Review by gparvin — COMMENTED** [review]
I had a couple minor comments I added...

**Review by gparvin — APPROVED** [review]
Thanks for the great contribution!

**Pull request merged**